### PR TITLE
Include MNE-ICALabel and PyTorch in Windows installers

### PR DESCRIPTION
**Pull request by hoechenberger**
New attempt after the failed #168

### CODE REVIEW

**Comment by hoechenberger** [comment]
Not sure if I'm doing something wrong with the package specs for Windows 🤔 the package name is:

```
win-64/pytorch-1.13.1-py3.10_cpu_0.tar.bz2
```
in the `pytorch` channel.

File: recipes/mne-python_1.3/construct.yaml [hunk 1]
```diff
@@ -75,10 +75,17 @@ specs:
...
   - autoreject =0.4.1
   - pyprep =0.4.2
   - openmeeg =2.5.5
+  # Deep learning
+  # We explicitly request the CPU version of the package, as the package size
+  # is only a fraction of the CUDA-enabled package (which also blows up NSIS
+  # for the Windows build).
+  # PyTorch for Windows will be pulled from the pytorch channel
+  # (see "channels" section below).
+  - pytorch=*=*cpu*
```
Comment (hoechenberger) [inline recipes/mne-python_1.3/construct.yaml:88]:
```suggestion
  - pytorch=*=cpu*  # [not win 
]
  - pytorch  # [win]
  - cpuonly  # [win]
```

File: recipes/mne-python_1.3/construct.yaml [hunk 2]
```diff
@@ -75,10 +75,20 @@ specs:
...
   - pyprep =0.4.2
   - openmeeg =2.5.5
+  # Deep learning
+  # We explicitly request the CPU version of the package, as the package size
+  # is only a fraction of the CUDA-enabled package (which also blows up NSIS
+  # for the Windows build).
+  # PyTorch for Windows will be pulled from the pytorch channel
+  # (see "channels" section below).
+  - pytorch=*=cpu*  # [not win 
+]
```
Comment (hoechenberger) [inline recipes/mne-python_1.3/construct.yaml:89]:
```suggestion
  - pytorch=*=cpu*  # [not win]
```

**Comment by larsoner** [comment]
Installation works but testing failed:
```
Trying to import MNE and all additional packages included in the installer
Traceback (most recent call last):
  File "D:\a\mne-installers\mne-installers\tests\test_imports.py", line 13, in <module>
    import mne_icalabel
  File "C:\Users\runneradmin\mne-python\1.3.0_1\lib\site-packages\mne_icalabel\__init__.py", line 3, in <module>
    from . import annotation  # noqa: F401
  File "C:\Users\runneradmin\mne-python\1.3.0_1\lib\site-packages\mne_icalabel\annotation\__init__.py", line 1, in <module>
    from .bids import mark_component, write_components_tsv
  File "C:\Users\runneradmin\mne-python\1.3.0_1\lib\site-packages\mne_icalabel\annotation\bids.py", line 7, in <module>
    from ..config import ICLABEL_LABELS_TO_MNE
  File "C:\Users\runneradmin\mne-python\1.3.0_1\lib\site-packages\mne_icalabel\config.py", line 1, in <module>
    from .iclabel import iclabel_label_components
  File "C:\Users\runneradmin\mne-python\1.3.0_1\lib\site-packages\mne_icalabel\iclabel\__init__.py", line 8, in <module>
    from .label_components import iclabel_label_components  # noqa: F401
  File "C:\Users\runneradmin\mne-python\1.3.0_1\lib\site-packages\mne_icalabel\iclabel\label_components.py", line 9, in <module>
    from .network import run_iclabel
  File "C:\Users\runneradmin\mne-python\1.3.0_1\lib\site-packages\mne_icalabel\iclabel\network.py", line 7, in <module>
    import torch
  File "C:\Users\runneradmin\mne-python\1.3.0_1\lib\site-packages\torch\__init__.py", line 128, in <module>
    raise err
OSError: [WinError 182] The operating system cannot run %1. Error loading "C:\Users\runneradmin\mne-python\1.3.0_1\lib\site-packages\torch\lib\shm.dll" or one of its dependencies.
```

**Comment by hoechenberger** [comment]
Bummer.....

**Comment by larsoner** [comment]
If you really want to go the extra mile you could borrow the `dependencies.exe` stuff from OpenMEEG to figure out what the DLL can't resolve

https://github.com/openmeeg/openmeeg/blob/993cabd4541720e7885b91e5dfaff1ff85c501ed/.github/workflows/build_and_test.yml#L476-L481

It should walk the dependencies and say what's missing

File: recipes/mne-python_1.3/construct.yaml [hunk 3]
```diff
@@ -75,11 +75,20 @@ specs:
...
   - pyprep =0.4.2
   - openmeeg =2.5.5
+  # Deep learning
+  # We explicitly request the CPU version of the package, as the package size
+  # is only a fraction of the CUDA-enabled package (which also blows up NSIS
+  # for the Windows build).
+  # PyTorch for Windows will be pulled from the pytorch channel
+  # (see "channels" section above and below).
+  - pytorch=*=cpu*  # [not win]
+  - pytorch  # [win]
```
Comment (hoechenberger) [inline recipes/mne-python_1.3/construct.yaml:90]:
```suggestion
  - pytorch=*=*cpu*  # [win]
```

**Pull request closed without merging**